src/realm/sync/noinst/client_impl_base.hpp

```diff
@@ -47,6 +47,8 @@ struct ServerEndpoint {
 private:
     auto to_tuple() const
     {
+        // Does not include server_mode because PBS and FLX connections can be
+        // multiplexed together. is_verified is not part of an endpoint's identity.
```

Comment:
PBS/FLX connections cannot be multiplexed together, but in practice they never will be because you have one SyncClient per app, and each app is always either all PBS or all FLX.

Reply:
Ah right. I'll fix the comment.